db/payment_icons.yml

```diff
@@ -285,8 +285,12 @@
 -
   name: swish
   label: Swish
-  group: other
+  group: bank_transfers
 -
+  name: ratepay
+  label: RatePay
+  group: wallets  
+ -
```

Comment:
I believe I found the issue. This `-` is indented one space. We need to remove this indentation.